pkgs/development/python-modules/vg/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, numpy
+, poetry-core
+, pythonOlder
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "vg";
+  version = "2.0.0";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "lace";
+    repo = "vg";
+    rev = "refs/tags/${version}";
+    hash = "sha256-ZNUAfkhjmsxD8cH0fR8Htjs+/F/3R9xfe1XgRyndids=";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+    setuptools
```

Comment:
```suggestion
```
I think we don't need setuptools to build.
https://github.com/lace/vg/blob/bece5191756b43378e882fd1fdf0ffa45a06e467/pyproject.toml#L43-L44

Reply:
Thanks, I tried building it without setuptools but it complaint that it is missing. I will try again later today, maybe it can be resolved with some patches.

Reply:
Yes, we can use `substituteInPlace` in `postPatch`.